File: 100-Days-Series/Day 26/SumOfNaturalNumbers.c
#include<stdio.h>
// Recursion method fo find sum of first n natural numbers.
int getSum(int num){
    if(num==1){
        return 1;// Base case
    }
    return num+getSum(num-1); // Recursion step
}
// Let's Start here
void main(){
    int num;
    printf("\nEnter a number : ");
    scanf("%d",&num);
    // Call the function and display result
    printf("\nSum of first %d natural numbers : %d",num,getSum(num));
}

File: 100-Days-Series/Day 26/SumOfNaturalNumbers.js

// Recursion(Arrow) method to find sum of first n natural numbers.
const getsum=(num)=>{
    if(num==1)
        return 1;// Base Case
    return num+getsum(num-1);
}
// Let's Start here
let num=100;
// Call the arrow function and display result
console.log("Sum of first "+num+" natural number : "+getsum(num));
